openmeetings-web/src/main/java/org/apache/openmeetings/web/room/wb/WbPanel.html

```diff
@@ -163,12 +163,12 @@
 		<div id="wb-tool-settings" class="wb-tool-settings ui-widget-content">
 			<div wicket:message="title:843" class="header ui-dialog-titlebar ui-widget-header ui-helper-clearfix ui-draggable-handle">
 				<span class="ui-dialog-title"><wicket:message key="843"/></span>
-				<button type="button" class="ui-button ui-corner-all ui-widget ui-button-icon-only ui-dialog-titlebar-close" wicket:message="title:85">
-					<span class="ui-button-icon ui-icon ui-icon-closethick"></span>
-					<span class="ui-button-icon-space"> </span>
-					<wicket:message key="85"/>
-				</button>
 			</div>
+			<button type="button" class="ui-button ui-corner-all ui-widget ui-button-icon-only ui-dialog-titlebar-close" wicket:message="title:85">
```

Comment:
This also can be bootstrap button :)

Reply:
yeah I think so too. Prob another Jira worth to review those dialogs